Add a new commands page and do some fixing/improvements (#373)
owncloud/docs-ocis: modules/ROOT/pages/maintenance/commands/commands.adoc

= Maintenance Commands
:toc: right

> :description: There are commands availabe to maintain the Infinite Scale instance. This document gives an overview of the available commands and links to the related service where they are implemented for more details.

Reviewer: ```suggestion
:description: There are commands availabe to maintain the Infinite Scale instance. This document gives an overview of the available commands and links to the related services where they are implemented for more details.
```

> :description: There are commands availabe to maintain the Infinite Scale instance. This document gives an overview about the available commands and links to the related service where they are implemented for more details.

Reviewer 2: ```suggestion
:description: There are commands availabe to maintain the Infinite Scale instance. This document gives an overview of the available commands and links to the related service where they are implemented for more details.
```

== Introduction

{description}

[NOTE]
====
* Calling a service in the examples is made via the single binary.
* Generic call: +
`ocis <service name> arguments options`.

> * This changes if services would be available as executable on their own, then it would be: +

Reviewer 2: ```suggestion
* This changes if some services are available as executable on their own, then it would be: +
```

`<service name> arguments options`. +
This could happen if services are developed which are not shipped as part of the single binary.
====

== Manage Expired Uploads

> Compared to unfinished uploads which are handled by the system, managing expired uploads can be a necessity as those files can pile up, blocking storage resources and need to be removed by command regularily. See the xref:{s-path}/storage-users.adoc#manage-unfinished-uploads[Manage Unfinished Uploads] section at the _Storage Users_ service for details.

Reviewer 2: ```suggestion
Compared to unfinished uploads which are handled by the system, managing expired uploads can be a necessity as those files can pile up, blocking storage resources and need to be removed by command regularly. See the xref:{s-path}/storage-users.adoc#manage-unfinished-uploads[Manage Unfinished Uploads] section at the _Storage Users_ service for details.
```

== Reindex a Space for Search

> In rare circumstances, it can be necessary to initiate indexing manually for a given space or user. Though the search service handles exception cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut-down of the service or because of a bug. The following command should *only* be run on explicit request and supervision of ownCloud support. See the xref:{s-path}/search.adoc#manually-trigger-re-indexing-a-space[Manually Trigger Re-Indexing a Space] section at the _Search_ service for details.

Reviewer: ```suggestion
In rare circumstances, it can be necessary to initiate indexing manually for a given space or user. Though the search service handles exception cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut-down of the service or because of a bug. Re-indexing should *only* be run on explicit request and supervision by ownCloud support. See the xref:{s-path}/search.adoc#manually-trigger-re-indexing-a-space[Manually Trigger Re-Indexing a Space] section at the _Search_ service for details.
```

> In rare circumstances, it can be necessary to initiate indexing manually for a given space or user. Though the search service handles exception cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut-down of the service or because of a bug. The following command should *only* be run on explicit request and supervision of ownCloud support. See the xref:{s-path}/search.adoc#manually-trigger-re-indexing-a-space[Manually Trigger Re-Indexing a Space] section at the _Search_ service for details.

Reviewer 2: There is no following command. I suppose that the command(s) is in the linked section.

Maybe say "The re-indexing command should..."
